values.yaml

```diff
@@ -908,11 +908,18 @@ database:
     sslmode: "disable"
   # The maximum number of connections in the idle connection pool per pod (core+exporter).
   # If it <=0, no idle connections are retained.
-  maxIdleConns: 100
+  maxIdleConns: 50
```

Comment:
please don't change the default value of `maxIdleConns` and `maxOpenConns`. It has been working fine and it is consistent with the goharbor/harbor.

Reply:
updated the README, and setback the default to 100 and 900